#goal definition
geometry_msgs/PoseStamped target_pose
---
#result definition
bool success
---
#feedback
bool is_possible_go
